packages/vc-http-api-test-server/config/index.js

```diff
@@ -0,0 +1,16 @@
+
+
+/**
+ * Configuration for running the test suite
+ */
+module.exports = [
```

Comment:
we probably want to wire these to CI, so that each vendor is shown as passing / failing on each PR, please create an issue to track this requested feature